docs/wiki/GitHub-Actions.md

```diff
@@ -1,68 +1,171 @@
 
-#### Create the repository
 
-Browse to the [AzOps Accelerator](https://github.com/azure/azops-accelerator), and select *Use this template*
 
-![Create the repository from the template](./Media/Actions/Template-Repository.png)
+## Prerequisites
 
-Select whether the owner will be an organization or user and provide a repository name.
+* [Create the Service Principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal)
 
-We'd recommended creating the new repository as *private*.
+* [Assign the permissions at the required scope (/)](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal)
+* Assign the Directory role permissions
 
-There is no need to include all branches as all latest stable changes reside within the main branch.
+* [Create project](https://docs.microsoft.com/en-us/azure/devops/organizations/projects/create-project?view=azure-devops&tabs=preview-page)
 
-![Configure the repository](./Media/Actions/Repository-Configuration.png)
+Please check if the Az and AzureAD modules are installed locally before executing these scripts. Alternatively, these command can be run within a Cloud Shell instance.
 
-#### Configure the secrets
+### Powershell command to create SPN:
+```powershell
+Connect-AzAccount
+$servicePrincipal = New-AzADServicePrincipal -Role Owner -Scope / -DisplayName AzOps
+Write-Host "ARM_TENANT_ID: $((Get-AzContext).Tenant.Id)"
+Write-Host "ARM_SUBSCRIPTION_ID: $((Get-AzContext).Subscription.Id)"
+Write-Host "ARM_CLIENT_ID: $($servicePrincipal.ApplicationId)"
+Write-Host "ARM_CLIENT_SECRET: $($servicePrincipal.Secret | ConvertFrom-SecureString -AsPlainText)"
+```
+### Powershell command to assign the Directory role permissions
+
+```powershell
+Install-Module -Name AzureAD
+Connect-AzureAD
+$servicePrincipal = Get-AzureADServicePrincipal -Filter "DisplayName eq 'AzOps'"
+$directoryRole = Get-AzureADDirectoryRole -Filter "DisplayName eq 'Directory Readers'"
+if ($directoryRole -eq $null) {
+    Write-Warning "Directory Reader role not found"
+}
+else {
+    Add-AzureADDirectoryRoleMember -ObjectId $directoryRole.ObjectId -RefObjectId $servicePrincipal.ObjectId
+}
+```
 
-Navigate to *settings* on the newly created repository, select the *Secrets* section.
+### Important Repo Link to refer
 
-Create the following repository secrets:
+Repo | Description
+-|-
+[AzOps Accelerator](https://github.com/Azure/AzOps-Accelerator.git) | This template repository is for getting started with the AzOps integrated CI/CD solution.
 
-![Configure the repository secrets](./Media/Actions/Repository-Secrets.png)
+### Two ways to configure AzOps
+- via Potal
+- Via command line script
 
-Select the *Options* sections, untick *Merge commits* and *Rebase merging*.
+### Configure AzOps via Portal
 
-![Configure the merge types](./Media/Actions/Merge-Types.png)
+- Import the above [Repo](https://github.com/Azure/AzOps-Accelerator.git) to new project.
```

Comment:
Since azops-accelerator is a template repository in GitHub, can we change to use that feature instead of importing the repository?
![image](https://user-images.githubusercontent.com/16622613/135085820-0f72b1dc-47a7-4da1-8cc7-fec95e831788.png)
https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template

Reply:
Agree, this change has been incorporated now.